### ISSUE TITLE
Rule blocked: CORERULES-5852

### ISSUE THREAD

**Issue by gerrycampion** [cdisc-org/cdisc-rules-engine#483]
**Links to related JIRA Tickets**
- https://jira.cdisc.org/browse/CORERULES-5852

**Rule Information**
- **Standard**: SDTMIG
- **Rule ID**: SD1109
- **Rule Description**: Exposure (EX) dataset should be included in every submission.

**Describe the bug**
Refer to SD1109 and corresponding negative test data.

```yaml
Rule Type: Domain Presence Check
Check:
  all:
    - name: EX
      operator: not_exists
```

The issue is that the Domain Presence Check creates a dataframe variable for each dataset that exists in the study. But we are looking for a domain that is missing from the study. So when the check fires an issue, it automatically includes `EX` as an Output Variable even though it does not exist in the dataframe.

**Error returned from Rule Engine**
```json
{
"error":"An unknown exception has occurred"
"message":"Output variables: ['EX'] not found in dataset"
}
```

**Expected behavior**
There needs to be a way to run this type of rule without an error. One possibility is to provide a default value of "Missing" when an output variable is missing.

**Comment by gerrycampion** [comment]
Duplicate of: https://github.com/cdisc-org/cdisc-rules-engine/issues/168
